README.md

```diff
@@ -25,6 +25,7 @@ First you will want to install a theme plugin through the Jenkins Update Center.
 
 * [Dark theme](https://github.com/jenkinsci/dark-theme-plugin)
 * [Dark theme (OS settings respected)](https://github.com/jenkinsci/dark-theme-plugin)
+* [Solarized theme](https://plugins.jenkins.io/solarized-theme/)
```

Comment:
If Dark Theme plugin has two list entries (one for each theme it implements), Solarized needs three: Light, Dark, and System.

Reply:
it could prob just be one for dark